pluma/pluma-window.c

```diff
@@ -3442,9 +3442,7 @@ show_notebook_popup_menu (GtkNotebook    *notebook,
 #endif
 	if (event != NULL)
 	{
-		gtk_menu_popup (GTK_MENU (menu), NULL, NULL,
-				NULL, NULL,
-				event->button, event->time);
+		gtk_menu_popup_at_pointer (GTK_MENU (menu), NULL);
```

Comment:
Why using different functions here?
Later you choose gtk_menu_popup_at_widget as replacement?

Reply:
here:

press the secondary mouse button in the tab:

![pluma-window1](https://user-images.githubusercontent.com/7734191/40308792-9e30a1dc-5d07-11e8-8519-339a1adfe78d.png)

later:

get the tab focus with [ctrl]+[shift]+[tab] and press the "properties" key in your keyboard:

![pluma-window2](https://user-images.githubusercontent.com/7734191/40308814-b3cb664e-5d07-11e8-8414-ea5c59beb86b.png)

the same behavior without this PR

Reply:
Ok, but isn't it more elegant to use gtk_menu_popup_at_widget for pointer too, if possible?
In result the menu opens always at the same place and the document title is always readable.
But it is OK for me if you don't like that....

Reply:
In the PRs of deprecated functions, I like to respect the original behavior

But I have no problem if you decide to change it in future PRs, I agree with your idea

Reply:
I would love to see this fixed too, but yeah this can be a new PR.